lib/matplotlib/colors.py

```diff
@@ -1718,10 +1718,14 @@ def inverse(self, value):
         def autoscale(self, A):
             # i.e. A[np.isfinite(...)], but also for non-array A's
             in_trf_domain = np.extract(np.isfinite(self._trf.transform(A)), A)
+            if not in_trf_domain.size:
```

Comment:
This changes this for everything that uses `autoscale`; is that OK?  I guess its not failing any tests, but our masked tests were not terribly complete.  

I find `not in_trf_domain.size` enigmatic - I assume it returns None if there was nothing to extract?